scan.go

```diff
@@ -146,78 +157,243 @@ func ReadRR(q io.Reader, filename string) (RR, error) {
 //              }
 //	}
 //
-// Comments specified after an RR (and on the same line!) are returned too:
+// Comments specified after an RR (and on the same line!) are
+// returned too:
 //
 //	foo. IN A 10.0.0.1 ; this is a comment
 //
-// The text "; this is comment" is returned in Token.Comment. Comments inside the
-// RR are returned concatenated along with the RR. Comments on a line by themselves
-// are discarded.
+// The text "; this is comment" is returned in Token.Comment.
+// Comments inside the RR are returned concatenated along with the
+// RR. Comments on a line by themselves are discarded.
+//
+// To prevent memory leaks it is important to always fully drain the
+// returned channel. If an error occurs, it will always be the last
+// Token sent on the channel.
+//
+// Deprecated: New users should prefer the ZoneParser API.
```

Comment:
Add maybe: See the comments in the ZoneParser API?

Reply:
The documentation for the two is mostly the same, what was your thinking here?